packages/experiments/src/components/FloatingPicker/BaseFloatingPicker.tsx

```diff
@@ -197,11 +198,24 @@ export class BaseFloatingPicker<T, P extends IBaseFloatingPickerProps<T>> extend
   }
 
   protected updateValue(updatedValue: string): void {
-    let suggestions: T[] | PromiseLike<T[]> = this.props.onResolveSuggestions(
-      updatedValue,
-      this.props.selectedItems
-    );
-    this.updateSuggestionsList(suggestions, updatedValue);
+    // Call onInputChanged
+    if (this.props.onInputChanged) {
+      (this.props.onInputChanged as (filter: string) => void)(updatedValue);
+    }
+
+    if (this.props.resolveDelay) {
```

Comment:
I don't think you need a timer, I believe you can do` this._onResolveSuggestions = this.async.debounce(this.onResolveSuggestions, this.props.resolveDelay) ` in onMounted which will do the same thing but be a little cleaner. @dzearing might know more

Reply:
Oh that looks promising. I think it should do the trick. I'll change to use this.